### PR TITLE
feat: integrate feeder package

### PR DESCRIPTION
**Pull request by awais-ansari**
[TNL-9509](https://openedx.atlassian.net/browse/TNL-9509)

- Integrate [feeder](https://feeder.sh/dashboard) package
- Email field will not visible to the users.
- The API payload will automatically carry user email.
- Feedback message is required.
- 
**Note**
- FEEDER_PROJECT_ID is required to enable the feedback module on stage and production.

Screenshot 2022-02-03 at 10 47 32 PM
Screenshot 2022-02-03 at 10 48 05 PM
Screenshot 2022-02-03 at 10 49 08 PM
Screenshot 2022-02-03 at 10 49 30 PM

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -20,3 +20,4 @@ REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=''
 SITE_NAME=localhost
 USER_INFO_COOKIE_NAME='edx-user-info'
+FEEDER_PROJECT_ID='61f123796593f20004018b2f'
```
Comment (awaisdar001) [inline .env.development:23]: you shouldn't put API keys here, these should be carried over using env settings in edx internals for MFEs. please remove these.

File: src/discussions/discussions-home/DiscussionsHome.test.jsx [hunk 2]
```diff
@@ -29,10 +29,16 @@ function renderComponent(location = `/${courseId}/`) {
 
 describe('DiscussionsHome', () => {
   beforeEach(async () => {
+    setConfig({
+      ...getConfig(),
+      FEEDER_PROJECT_ID: 'abc123',
```
Comment (awaisdar001) [inline src/discussions/discussions-home/DiscussionsHome.test.jsx:34]: is that a test ID? `abc123` => `test-id` would be more readable.

**Review by awaisdar001 — CHANGES_REQUESTED** [review]
Great job for integrating the feedback tool in such a tight timeline.

Code quality is excellent 👍

File: src/components/Feedback/constants.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const FeedbackTypes = ['Review', 'Bug', 'Idea'];
+export const primaryColor = '#00262b';
```
Comment (mehaknasir) [inline src/components/Feedback/constants.js:2]: do we have this color code available in paragon? can we use that from there?

File: .env.development (hunk 1 above)
Comment (awais-ansari) [reply .env.development:23]: Sure. I'll remove it from this MEF.

File: src/discussions/discussions-home/DiscussionsHome.test.jsx (hunk 2 above)
Comment (awais-ansari) [reply src/discussions/discussions-home/DiscussionsHome.test.jsx:34]: Yes. It is a test ID. I'll update it.

File: src/components/Feedback/constants.js (hunk 3 above)
Comment (awais-ansari) [reply src/components/Feedback/constants.js:2]: We can only pass the color CSS value such as HEX, RGB, and paragon provides string classes such as `text-primary-500` which is not compatible with it. I tried this solution too.

**Review by awaisdar001 — APPROVED** [review]

**Review by mehaknasir — APPROVED** [review]

**Pull request merged**